tests/components/mqtt/test_fan.py

```diff
@@ -291,54 +295,48 @@ async def test_controlling_state_via_topic_no_percentage_topics(hass, mqtt_mock)
     async_fire_mqtt_message(hass, "preset-mode-state-topic", "freaking-high")
```

Comment:
Can we go with something like `auto`, `smart`, `whoosh`, etc so there isn't any future confusion here?

Reply:
I have updated preset_modes in `test_fan.py` to be aligned with the documentation, so there is no confusion this is about speeds.